.travis.yml

```diff
@@ -1,49 +1,42 @@
-language: android
-jdk:
-    - oraclejdk8
+sudo: required
 
+language: android
+jdk: oraclejdk8
+
+services:
+  - docker
+  
+env:
+ - TEST_SUITE=unit
+ - TEST_SUITE=instrumentation
+  
 android:
   components:
-    # Uncomment the lines below if you want to
-    # use the latest revision of Android SDK Tools
-    - tools
-    - platform-tools
-    - tools
-
-    # The BuildTools version used by your project
-    - build-tools-26.0.1
-
-    # The SDK version used to compile your project
-    - android-22
-    - android-26
-
-    # Additional components
-    - extra-google-google_play_services
-    - extra-google-m2repository
-    - extra-android-m2repository
-    - addon-google_apis-google-24
-
-    # Specify at least one system image,
-    # if you need to run emulator(s) during your tests
-    - sys-img-armeabi-v7a-android-22
-
-
+    - $(if [ $TEST_SUITE == "instrumentation" ]; then echo "tools,platform-tools,tools,build-tools-26.0.1,android-22,android-26,extra-google-google_play_services,extra-google-m2repository,extra-android-m2repository,addon-google_apis-google-24,sys-img-armeabi-v7a-android-22"; fi)
```

Comment:
can we make the echo multiline?

Reply:
Currently I create a `script.sh` which reads the `BUILD_TOOLS_VERSION` and `SDK_VERSION` and replace the given numbers.
These script return then all components..